backend/molgenis-emx2/src/main/java/org/molgenis/emx2/utils/JavaScriptUtils.java

```diff
@@ -15,11 +16,13 @@ public class JavaScriptUtils {
           .option("engine.WarnInterpreterOnly", "false")
           .build();
 
-  private JavaScriptUtils() {
-    // hide constructor
+  private final IdGenerator idGenerator;
```

Comment:
there will be only one implementation with singleton idGenerator. Please make this static so all these refactorings can be undone